lighthouse-core/report/templates/report-template.html

```diff
@@ -247,8 +249,14 @@ <h2 class="config-section__title">Blocked URL Patterns</h2>
 {{/each}}
 
 <script>
-window.addEventListener('DOMContentLoaded', _ => {
-  new LighthouseReport(window.lhresults);
+(function(exports) {
```

Comment:
okay on the one hand we have 
![image](https://cloud.githubusercontent.com/assets/39191/22721186/060386ea-ed64-11e6-8cac-7b7d763eac96.png)
but then here we have
![image](https://cloud.githubusercontent.com/assets/39191/22721219/26e2ba52-ed64-11e6-8608-fc4105f8c38c.png)

so yeah im pretty confused as well.